incubator/postgres-operator/0.0.1/Chart.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v2
+kubeVersion: ">=1.16.0-0"
+name: postgres-operator
+version: 0.0.1
+upstream_version: 1.6.2
+appVersion: "auto"
+description: Postgres Operator creates and manages PostgreSQL clusters running in Kubernetes
+type: application
+deprecated: false
+home: https://github.com/truecharts/apps/tree/master/incubator/postgres-operator
+icon: https://truecharts.org/_static/img/postgres-operator-icon.png
+keywords:
+  - database
+  - HA
+  - postgresql
+  - SQL
+  - postgres
+  - operator
+  - cloud-native
+  - patroni
+  - spilo
+sources:
+  - https://github.com/zalando/postgres-operator
+dependencies:
+  - name: common
+    repository: https://truecharts.org/
+    version: 2.0.2
+    # condition:
+  - name: postgres-operator-ui
+    repository: https://raw.githubusercontent.com/zalando/postgres-operator/master/charts/postgres-operator-ui/
```

Comment:
You are using as external dependency but you are adding bellow the tgzs - is this intentional for truenas?

Reply:
Yes, both are required.
(and actually when you do this yourself, Helm will fetch that tgz too, after you fetch the dependency ;-) )